File: 2. Arrays 101/0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        numsLength = len(nums) - 1
        while (numsLength >= 0):
            if (nums[numsLength] == val):
                nums.pop(numsLength)
            numsLength -= 1
        return len(nums)
        

File: Difficulty/A. Easy/0027-remove-element/0027-remove-element.c
int removeElement(int* nums, int numsSize, int val) {
    int notValIndex = 0;

    for (int i = 0; i < numsSize; i++) {
        if (nums[i] != val) {
            nums[notValIndex] = nums[i];
            notValIndex++;
        }
    }
    
    return notValIndex;
}
